### DISCUSSION TITLE
CORS error when load image in edit mode

### DISCUSSION THREAD

**Discussion by iranandroid** [filamentphp/filament#3607] [Help]
Hi everyone.
I have CORS error when load image in edit mode.

![image](https://user-images.githubusercontent.com/20900080/185879728-66759a0d-775b-4c21-b932-ce38c4da47fd.png)

![image](https://user-images.githubusercontent.com/20900080/185879820-22db8ab2-d084-4904-a835-3516489aad37.png)

How to fix this error?

**Comment by jigujigu** [comment]
Hi Everyone, 

I have same problem too..

**in create mode**
![image](https://user-images.githubusercontent.com/6214964/188352625-2bac525c-326c-4830-870e-adb0b0c8cd77.png)

**Image is successfully saved.**
![image](https://user-images.githubusercontent.com/6214964/188352959-f14ac8c7-b665-41b5-bcee-e8bd3761257b.png)

**in view mode** Image can't show-up.
![image](https://user-images.githubusercontent.com/6214964/188353017-b969632a-94d8-4042-a6ac-d46e0a2f569c.png)

**This is my form function :**
```
    public static function form(Form $form): Form
    {
        return $form
            ->schema([

                Forms\Components\FileUpload::make('picture')
                    ->label('picture')
                    ->directory('picture')
                    ->image()
                    ->required(),

                Forms\Components\Fieldset::make('Label')
                    ->schema([
                        Forms\Components\TextInput::make('name')->required(),
                        Forms\Components\TextInput::make('code')->required(),
                    ])
                    ->columns(3)
            ]);
    }
```


**And this is my config\cors.php**

```
<?php
return [
    'paths' => ['api/*', 'sanctum/csrf-cookie','web/*'],
    'allowed_methods' => ['*'],
    'allowed_origins' => ['*'],
    'allowed_origins_patterns' => [],
    'allowed_headers' => ['*'],
    'exposed_headers' => [],
    'max_age' => 0,
    'supports_credentials' => false,
];

```

Please help.. thank you.

**Reply by danharrin** [reply]
Your APP_URL is wrong in .env

**Reply by jigujigu** [reply]
Ok... 
Thanks it's work now.

I update the .env file with these line :

```
APP_URL=http://127.0.0.1:8000
FILESYSTEM_DISK=public
```

and make a link for the directory
`php artisan storage:link`

**Reply by cyberwinmesi** [reply]
Yep, `APP_URL` fixed it on my end as well (because of the missing `www.`) Thanks for that!
